apps.awesim.org/apps/myjobs/templates/Basic_Q-Chem_Job_Pitzer/sample.sh

```diff
@@ -0,0 +1,29 @@
+#PBS -N ondemand/sys/myjobs/basic_qchem
+#PBS -l walltime=0:59:00
+#PBS -S /bin/csh
+#PBS -j oe
+#PBS -l nodes=1:ppn=1
+#
+# This is a sample script for running a basic Q-Chem job.
+# The only thing you need to modify is 'sample' here:
+#
+setenv JOBNAME sample
+#
+# replace 'sample' with the actual name of your input file.
+#
+module purge
+module load intel/18.0.4 mvapich2/2.3 qchem/5.1.1
+
+# copy the contents to TMPDIR
+cp $PBS_O_WORKDIR/* $TMPDIR
+cd $TMPDIR
+
+# QChem guide at
+#   http://www.q-chem.com/qchem-website/manual/qchem43_manual/sect-running.html
+
+# save input files:
+# qchem -save $JOBNAME.inp $JOBNAME.out $JOBNAME
+# do not save input files:
+qchem $JOBNAME.inp $JOBNAME.out
```

Comment:
This template would be a better demonstration of HPC if you highlight `qchem`'s ability to run in either a multithreading or MPI mode:

"""
To run Q-Chem calculation with OpenMP threads specify the number of threads ( nthreads ) using qchem command option -nt. Since each thread uses one CPU core, you should not specify more threads than the total number of available CPU cores for performance reason. When unspecified, the number of threads defaults to 1 (serial calculation).

qchem -nt nthreads infile outfile
qchem -nt nthreads infile outfile save
qchem -save -nt nthreads infile outfile save
Similarly, to run parallel calculations with MPI use the option -np to specify the number of MPI processes to be spawned.

qchem -np n infile outfile
qchem -np n infile outfile savename
qchem -save -np n infile outfile savename
where $n$ is the number of processors to use. If the -np switch is not given, Q-Chem will default to running locally on a single node.
"""

http://www.q-chem.com/qchem-website/manual/qchem43_manual/sect-running.html

You would then also need to update the value for PPN to some multiple of 40.

Reply:
Good point. Just changed qchem template to one with MPI.